src/meta/cycleData/link/link.ts

```diff
@@ -0,0 +1,37 @@
+import { CountryIso } from 'meta/area'
+
+export type Link = {
+  readonly id: number
+  readonly uuid: string
+  countryIso: CountryIso
+  link: string
+  location: LinkLocation
+  props: LinkProps
+  target: string
+}
+
+export type LinkLocation = {
+  column: string
+  entityProps: Record<string, string>
+  id: number
+  table: string
+}
+
+export type LinkProps = {
```

Comment:
Should `LinkProps` be defined before usage at line `9`?

Reply:
Reordered it 👍